app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -15,9 +15,14 @@ def show
     def update
       clear_invalid_steps!
       result = Idv::HowToVerifyForm.new.submit(how_to_verify_form_params)
+      if how_to_verify_form_params[:selection] == []
+        sendable_form_params = Hash.new
```

Comment:
I feel like we only use `Hash.new` when setting a default value like `Hash.new(0)`
```suggestion
        sendable_form_params = {}
```

Reply:
This is helpful. I wasn't totally sure of the benefit of one over the other.